superset-frontend/src/SqlLab/components/SouthPane.tsx

```diff
@@ -83,33 +77,34 @@ const StyledPane = styled.div`
   }
 `;
 
-export class SouthPane extends React.PureComponent {
-  constructor(props) {
-    super(props);
-    this.southPaneRef = React.createRef();
-    this.switchTab = this.switchTab.bind(this);
-  }
+function SouthPane({
+  editorQueries,
+  latestQueryId,
+  dataPreviewQueries,
+  actions,
+  activeSouthPaneTab = 'Results',
+  height,
+  databases,
+  offline = false,
+  displayLimit,
+}: SouthPanePropTypes) {
+  const innerTabContentHeight = height - TAB_HEIGHT;
+  const southPaneRef: any = useRef();
```

Comment:
more specific type here?

Reply:
I forget, is there a preference between createRef and useRef?